Geometry_Engine/Query/IsClosed.cs

```diff
@@ -55,10 +55,20 @@ public static bool IsClosed(this Line line, double tolerance = Tolerance.Distanc
 
         /***************************************************/
 
-        [NotImplemented]
+        public static bool IsClosed(this Ellipse ellipse, double tolerance = Tolerance.Distance)
+        {
+            return true;
```

Comment:
I don't see what this has to do with the creation of methods related to the Planar surface.

It looks to me as if this has been sneaked in as part of this PR, without any issue or discussion around it. I hope the engine owners are aware and happy with this?

Reply:
My bad. That exists because `Create.PlanarSurface` uses two methods: `IsClosed` and `IsPlanar`, which lack for some of the geometry types. Ellipse is one of them, and it was required for the `Create` method to work.

You can add a unit test if you don't feel safe with the method 😄

Reply:
Confirmed. 🙈

Reply:
That explains it nicely - I just wanted to make sure no one was trying to use @IsakNaslundBh good tempered nature to sneak things in sneakily :wink: